lib/msal-common/src/authority/Authority.ts

```diff
@@ -552,13 +552,15 @@ export class Authority {
                 : [];
             if (metadata.length === 0) {
                 // If no metadata is returned, authority is untrusted
+                this.logger.verbose("The network request completed successfully, but no metadata was returned.");
                 return null;
             }
             match = Authority.getCloudDiscoveryMetadataFromNetworkResponse(
```

Comment:
I think you should add some logging in this method as well.

Reply:
I thought we decided that getCloudDiscoveryMetadataFromNetworkResponse doesn't need logging since it just contains a for loop?